node/tests/integration/main.rs

```diff
@@ -582,11 +582,10 @@ async fn assert_slot_status_call(
     let game_account_storage = account.storage();
 
     let remaining_balance = slot_data.player_balance() - slot_data.highest_bet();
-    let slot_index = 68;
 
     // check player balance
     assert_eq!(
-        game_account_storage.get_item(slot_index),
+        game_account_storage.get_item(PLAYER_BALANCE_SLOT),

```

Comment:
Actually, i meant to include constants for all the slot indexes used in the integrations test, Not just the ones which are used in call test.